docs/guide/deployment/resource-directory.md

```diff
@@ -227,14 +227,30 @@ This will be served by HTTP Gateway API as defined [here](https://github.com/plg
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
 | `publicConfiguration.caPool` | string | `File path to root CA which was used to sign coap-gw certificate.` | `""` |
-| `publicConfiguration.tokenURL` | string | `URL where user can get OAuth token via implicit flow.` | `""` |
-| `publicConfiguration.authorizationURL` | string | `URL where user can get OAuth authorization code for the device.` | `""` |
 | `publicConfiguration.ownerClaim` | string | `Claim used to identify owner of the device.` | `"sub"` |
 | `publicConfiguration.signingServerAddress` | string | `Address of ceritificate authority for plgd-dev/sdk.` | `""` |
 | `publicConfiguration.cloudID` | string | `Cloud ID which is stored in coap-gw certificate.` | `""` |
 | `publicConfiguration.cloudURL` | string | `Cloud URL for onboard device.` | `""` |
 | `publicConfiguration.cloudAuthorizationProvider` | string | `OAuth authorization provider for onboard device.` | `""` |
-| `publicConfiguration.defaultCommandTimeToLive` | string | `Exposes default command time to live for CreateResource, RetrieveResource, UpdateResource, DeleteResource, and UpdateDeviceMetadata commands when it is zero value in the request. 0s - means forever.` | `"0s"` |
+| `publicConfiguration.defaultCommandTimeToLive` | string | `Exposes default command time to live for CreateResource, RetrieveResource, UpdateResource, DeleteResource, and UpdateDeviceMetadata commands when it is not set in the request. 0s - means forever.` | `"0s"` |
```

Comment:
Don't document items that are not used at all. Public configuration requires just client id, redirect url audience, scope and authority.